index.ts

```diff
@@ -0,0 +1,68 @@
+import express, { Express, Request, Response } from "express";
+import bodyParser from "body-parser";
+import { auctionPaid } from "./types/interfaces";
+import { submitPayment } from "./submitPayment";
+import { saveReciept } from "./saveReceipt";
+import { searchPaid } from "./searchPaid";
+import cors from "cors";
+
+const app: Express = express();
+app.use(bodyParser.json());
+app.use(cors());
+app.use(
+    bodyParser.urlencoded({
+        extended: true,
+    })
+);
+
+const port = 3000;
+
+//pay and save reciept
+app.post(
+    "/submitPayment",
+    async(
+        req: Request<unknown, unknown, auctionPaid, unknown>,
+        res: Response
+    ) =>{
+        try{
+
+            const response = await submitPayment();
+
+            const response1 = await saveReciept(
+                req.body.item_id,
+                req.body.item_reciept
+            );
+            res.sendStatus(response,response1);
```

Comment:
you only need to send the response from `saveReceipt`, although you should check to see if `submitPayment` returns true before you ask to save the receipt